parser/http.go

```diff
@@ -52,7 +64,18 @@ func parseHTTPEntry(parseHTTP *parsetypes.HTTP, filter filter, retVals ParseResu
 		uri = uri[:maxIndex]
 
 		// at this point, the URI should be parsed down to just an FQDN
+		// set fqdn or skip log entry if still unable to parse value
+		if uri == "" {
```

Comment:
Please remove this check in order to make sure we don't drop anything we would have previously accepted as valid. 

It looks like both the URI and Host fields may be unset in valid Zeek HTTP records. 
I believe this happens when we capture an HTTP request without a host header and with the URI just set to `/`. 

Here is an example Zeek log where something like this occurred:
```
#fields ts      uid     id.orig_h       id.orig_p       id.resp_h       id.resp_p       trans_depth     method  host    uri     referrer        version user_agent      request_body_len response_body_len       status_code     status_msg      info_code       info_msg        tags    username        password        proxied orig_fuids      orig_filenames  orig_mime_types  resp_fuids      resp_filenames  resp_mime_types
1518768200.830914      Cfqzwb2r0r1VWZhsOe      10.55.100.106   60960   23.63.152.72    80       1       -       -       -       -       1.0     -       0       218     408     Request Time-out        -       -       (empty) -       -       -       -       -       -FC0oK8vYEC1rxh8th       -       text/html
```
---

Perhaps we should set the fqdn to be the destination IP address when this occurs so we don't index it as a connection to `""`. This would likely be better addressed in a separate ticket.